spec/acceptance/nodesets/debian-610-x64.yml

```diff
@@ -0,0 +1,12 @@
+HOSTS:
+  debian-610-x64:
+    roles:
+      - master
+    platform: debian-6-amd64
+    box: puppetlabs/debian-6.0.10-64-nocm
+    box_url: https://vagrantcloud.com/puppetlabs/boxes/debian-6.0.10-64-nocm
```

Comment:
I'm okay with us having debian 6 tests, but I'm not real big on considering it a supported platform, given that it's two versions out of date now. Do you need it or is this added for the sake of being thorough?